SwiftLeeds/Views/My Conference/AnnouncementCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  AnnouncementCell.swift.swift
+//  SwiftLeeds
+//
+//  Created by Matthew Gallagher on 25/06/2022.
+//
+
+import SwiftUI
+
+struct AnnouncementCell: View {
+    let label: String
+    let value: String
+    let valueIcon: String
+    let gradientColors: [Color]
+
+    var body: some View {
+        HStack {
+            Text(label)
+                .fontWeight(.medium)
+            Spacer()
+            Image(systemName: valueIcon)
+            Text(value)
+                .fontWeight(.medium)
+        }
+        .padding(12)
+        .foregroundColor(.white)
+        .frame(height: 65)
```

Comment:
Setting an explicit height like this breaks dynamic type compatibility.

Let's avoid setting hard heights on containers and instead prefer setting sizes on the images inside these rows.

So in this case the 65 would be on the Image, not the stack

Reply:
I took this height from the design as wasn't sure what the intent was.  We can't use the image as we'd want that to also size with dynamic text, but could use padding instead of a fixed size.

Reply:
Adding vertical padding instead to resolve this